website/docs/reference/node-selection/methods.md

```diff
@@ -46,6 +46,13 @@ The `source` method is used to select models that select from a specified [sourc
   $ dbt run --select source:snowplow+    # run all models that select from Snowplow sources
   ```
 
+### The "resource_type" method
+The `resource_type` method is used to select nodes of a particular type (`model`, `source`, `exposure`, etc). This is similar to the `--resource-type` flag used by the [`dbt ls` command](/reference/commands/list).
+
+  ```bash
+  $ dbt build --select +resource_type:exposure    # build all resources upstream of exposures
+  $ dbt list --select +resource_type:test    # list all tests in your project
```

Comment:
The ["plus" operators](https://docs.getdbt.com/reference/node-selection/graph-operators#the-plus-operator) (`+`) here seem extraneous, so I'm going to pull them out:
```suggestion
  $ dbt build --select resource_type:exposure    # build all resources upstream of exposures
  $ dbt list --select resource_type:test    # list all tests in your project
```